### DISCUSSION TITLE
Install the driver in OKD Openshift

### DISCUSSION THREAD

**Discussion by dborgi** [ceph/ceph-csi#4080] [Q&A]
have an OKD 4.13 cluster and I need to install the logging with the Redhat Logging and ElasticSearch operators, and I want to use Ceph as dynamic storage, I installed version 18.2.0 but I am not being able to do bound, I would appreciate it if you could help me, thanks

![image](https://github.com/ceph/ceph-csi/assets/48066051/3161abbf-50fa-44f4-a6d2-fecbd49447f6)

**Comment by nixpanic** [comment]
I recommend you have a look at the Rook operator. It has very complete [documentation on how to install and (automatically) configure Ceph for OpenShift](https://rook.io/docs/rook/v1.12/Getting-Started/ceph-openshift/).

If you have issues after that, you'll need to share more details on the problem you are experiencing.

**Reply by dborgi** [reply]
thanks for the reply, to add, my ceph cluster is installed outside of the okd cluster.

**Reply by Madhu-1** [reply]
if you have ceph cluster outside of OKD, you can use external-ceph cluster in Rook https://rook.io/docs/rook/v1.12/CRDs/Cluster/external-cluster/, it allows you to use external ceph cluster.
